STYLE.md

```diff
@@ -649,6 +649,40 @@ A common mistake with refs is using them to pass data back to a parent component
 
 There are several ways to use and declare refs and we prefer the [callback method](https://reactjs.org/docs/refs-and-the-dom.html#callback-refs).
 
+In addition, all refs should be declared in the constructor and null-initialized, rather than inline. This makes it easier to quickly see what refs are declared in the component:
+
+```jsx
+class BadRefComponent extends Component {
+    constructor(props) {
+        super(props);
+    }
+    
+    render() {
+        return (
+            <View
+                // Bad: Ref is declared inline
+                ref={el => this.myRef = el}
+            />
+        )
+    }
+}
+
+
+class GoodRefComponent extends Component {
+    constructor(props) {
+        super(props);
+        
+        // Good: Ref is declared in the constructor
+        this.myRef = null;
```

Comment:
Why `null` and not `undefined`? Does it make any difference?

Reply:
No big difference, and I have no preference.